client/app/license/order/license-order.html

```diff
@@ -217,19 +217,16 @@ <h2 data-translate="license_order_step4_title"></h2>
                                     data-ng-if="agoraEnabled"
                                     data-translate="license_order_step4_contracts_agora_ask">
                             </strong>
-                            <table class="table table-hover">
-                                <tbody>
-                                    <tr data-ng-repeat="contract in durations.details[selected.duration].contracts">
-                                        <td>
-                                            <a target="_blank"
-                                               rel="noopener"
-                                               data-ng-href="{{ contract.url }}"
-                                               data-ng-bind="contract.url">
-                                            </a>
-                                        </td>
-                                    </tr>
-                                </tbody>
-                            </table>
+                            <span data-ng-repeat="contract in durations.details[selected.duration].contracts track by $id">
```

Comment:
Wouldn't a `div` be better ? To have the links one above the other ?

Reply:
Or an unordered list `<ul>`:)